src/com/company/Utility/UtilityOperations.java

```diff
@@ -0,0 +1,22 @@
+package com.company.Utility;
+import com.company.Player;
+import com.company.Team;
+
+public class UtilityOperations {
+    private UtilityOperations() {}
+
+    public static Player[] swapPlayer(Player strikerBatsman, Player nonStrikerBatsman)
```

Comment:
Java is a pass by reference language. if you swap objects of strikerBatsman and nonStrikerBatsman in this method, this variables will get swapped in the parent method.